config/languages.yml

```diff
@@ -44,9 +44,10 @@ default: &default
   es: Español
   et: Eesti
   eu: Euskara
+  fa: Persian (فارسی)
+  fj: Na Vosa Vakaviti
   ff: Fulfulde
   fi: Suomi
-  fj: Na Vosa Vakaviti
```

Comment:
Please restore this line to how it was, thanks!

Reply:
@BaseMax Still this one to do, thanks!

Reply:
I think solved. 👌😬